modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -14,38 +14,6 @@ Automatically scale an Ingress Controller to dynamically meet routing performanc
 . You have the Custom Metrics Autoscaler Operator installed.
 
 .Procedure
-. Create a project in the `openshift-ingress-operator` namespace by running the following command:
```

Comment:
This step should be re sentence as we should  be in the `openshift-ingress-operator` namespace before running the below steps. It should be 
`Switch to the openshift-ingress-operator namespace by running the following command:
$ oc project openshift-ingress-operator

Reply:
gotcha, thank you. I will add this to the prerequisites section actually